var/spack/repos/builtin/packages/nn-c/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class NnC(AutotoolsPackage):
+    """nn: Natural Neighbours interpolation. nn is a C code
+    for Natural Neighbours interpolation of 2D scattered data.
+    It provides a C library and a command line utility nnbathy."""
+
+    homepage = "https://github.com/sakov/nn-c"
+    git      = "https://github.com/sakov/nn-c.git"
+
+    version('master', branch='master')
+    version('1.8.62', commit='343c7784d38d3270d75d450569fc0b64767c37e9')
```

Comment:
@ilbiondo It's the 86th minor 2nd patch apparently, not the 8th minor 62nd patch :laughing: 
```suggestion
    version('1.86.2', commit='343c7784d38d3270d75d450569fc0b64767c37e9')
```